src/modals/send-modal/send-modal.form.tsx

```diff
@@ -18,7 +19,7 @@ export const sendModalValidationSchema: SchemaOf<SendModalFormValues> = object()
   token: object().shape({}).required(makeRequiredErrorMessage('Asset')),
   receiverPublicKeyHash: string()
     .when('transferBetweenOwnAccounts', (value: boolean, schema: StringSchema) =>
-      value ? schema : schema.required(makeRequiredErrorMessage('To'))
+      value ? schema : tokenAddressValidation
```

Comment:
`tokenAddressValidation` checks if an address is a valid `KT` address, but wallets are starts with `tz`.  
Create a new validation for this

Reply:
take a look at extension validation and improve it